src/Test/Utilities/Desktop/TestResource.resx

```diff
@@ -123,7 +123,7 @@
 #pragma warning disable 414, 3021
```

Comment:
In this case some errors and warnings nevertheless will be displayed in Error List:
```
#error: 'Error message'
#warning: 'Warning message'
The 'l' suffix is easily confused with the digit '1' -- use 'L' for clarity	Miscellaneous Files
The 'l' suffix is easily confused with the digit '1' -- use 'L' for clarity	Miscellaneous Files
The 'l' suffix is easily confused with the digit '1' -- use 'L' for clarity	Miscellaneous Files
Empty switch block
```
And this may be awkward for compiling.